src/sentry/event_manager.py

```diff
@@ -2772,36 +2772,57 @@ def save_attachment(
 
         logger.exception("Missing chunks for cache_key=%s", cache_key)
         return
+    from sentry import ratelimits as ratelimiter
 
-    file = EventAttachment.putfile(project.id, attachment)
-
-    EventAttachment.objects.create(
-        # lookup:
-        project_id=project.id,
-        group_id=group_id,
-        event_id=event_id,
-        # metadata:
-        type=attachment.type,
-        name=attachment.name,
-        content_type=file.content_type,
-        size=file.size,
-        sha1=file.sha1,
-        # storage:
-        file_id=file.file_id,
-        blob_path=file.blob_path,
+    is_limited, num_requests, reset_time = ratelimiter.backend.is_limited_with_value(
+        key="event_attachment.save",
+        limit=options.get("sentry.save-event-attachments.project-per-5-minute-limit"),
+        project=project,
+        window=60,
```

Comment:
description says 5 minutes, but the window is 60 seconds

Reply:
Thank you